launch-yocto.sh

```diff
@@ -233,6 +233,10 @@ generate_report() {
   mv ../seapath-themes/logo.png themes/sfl.png
   # Change contact mailing list to seapath SFL mailing list
   sed -i 's/contact@savoirfairelinux/seapath@savoirfairelinux/g' test-report.adoc
+        
+  # Move openlab part
+  mv ../openlab/doc/images/setup_lat.png doc/
+  mv ../openlab/test-report.adoc ../openlab/test-report-skeleton.adoc .
```

Comment:
use absolute path instead of relavite ones. Like `"${WORK_DIR}/ci/openlab/......`
Also change it on line 233 if you can